content/300-guides/050-database/500-planetscale-guide.mdx

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Suggestion to rename file to `content/300-guides/050-database/500-planetscale.mdx` so resulting URL is `/docs/guides/database/planetscale`.

Reply:
I've renamed to `500-using-prisma-with-planetscale.mdx`, which matches the format of the other guides in that section

Reply:
I thought we would move everything else into another section via the IA effort?